src/components/limited-restoration-panel/LimitedRestorationPanel.vue

```diff
@@ -60,6 +60,10 @@ export default class LimitedRestorationPanel extends Mixins(DateMixin) {
       this.selectMonths = 'customMonths'
       this.numberOfMonths = draftMonths
       this.setCustomMonths(this.numberOfMonths)
+      // Emit validation on load
+      if (this.numberOfMonths > this.maxNumberOfMonths || this.numberOfMonths < 1) {
+        this.monthsValid(false)
+      }
```

Comment:
Yes, I think it's fine for a component to emit validation on load.

One thing to watch for: this assumes the data is current when the component loads. In some cases, the initial data is replaced when, say, a draft is finished loading -- so the component also needs to react to that. I don't know if that's the case here; if it tests OK then you're probably fine.

Reply:
I tested it and seems to be working fine! That's a good point though. I'll keep that in mind.